src/Mod/PartDesign/App/FeatureFillet.cpp

```diff
@@ -56,6 +56,9 @@ Fillet::Fillet()
     ADD_PROPERTY(Radius,(1.0));
     Radius.setUnit(Base::Unit::Length);
     Radius.setConstraints(&floatRadius);
+    ADD_PROPERTY_TYPE(UseAllEdges,(false),"Base", App::Prop_None,
+      "Fillet all edges if true, else if false use only those edges in Base property\n"
+      "If true, then this overrides any edge changes made to the Base property or in the dialog.\n");

```

Comment:
```suggestion
    ADD_PROPERTY_TYPE(UseAllEdges,(false),"Base",App::Prop_None,
      "Fillet all edges if true, else use only those edges in Base property.\n"
      "If true, this overrides any edge changes made to the Base property or in the dialog.\n");

```

Reply:
Okay, I like that wording better.  I need spaces after the commas, too.

Reply:
Thanks for making the changes. I have seen that when using a macro, commas are often not used. However, it seems they are used in some other places, so that's okay I suppose.